client/js/models/Header.js

```diff
@@ -1,9 +1,9 @@
-module.exports = Object.assign( {}, require('./__proto__.js'), {
-
+module.exports = Object.assign({}, require('./__proto__.js'), {
     data: [
-        'home',
-        'projects',
-        'contact'
+        'about us',
+        'where to find us',
+        'future days farm',
+        'the blog',
+        'our offerings'
     ]
-
-} )
+})
```

Comment:
Sorry to be so pedantic, but I just want you to keep pace with the many things I've learned over the past few months.

Add a .babelrc to the project like so: https://github.com/FutureDaysSoftware/MaricannPortal/blob/configRefinement/.babelrc

Babel let's one write really fancy ( syntactic sugar ) javascript, and transforms it to older javascript to be loaded any browser.  Object.assign is now rarely needed with spread syntax: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax

You can do:

```
module.exports = { ...require('./__proto__.js'),  data: [  'string1', 'str2' ] }
```